### PR TITLE
Remove indirect, multidimensional, and bareword_filehandles from v5.36 feature bundle

### PR DESCRIPTION
**Pull request by rjbs**

### CODE REVIEW

File: pod/perldelta.pod [hunk 1]
```diff
@@ -33,6 +33,15 @@ Attempting to specify a list after C<for my> was previously a syntax error.
 
 For more detail see L<perlsyn/Compound Statements>.
 
+=head2 C<use v5.36> now disables bareword_filehandles and indirect
+
+The :5.36 bundle, loaded by C<use v5.36>, no longer includes
```
Comment (leonerd) [inline pod/perldelta.pod]: Should `:5.36` be annotated by `C<...>` bracket?

File: pod/perlfunc.pod [hunk 2]
```diff
@@ -4903,7 +4903,11 @@ An older style is to use a bareword as the filehandle, as
 
 Then you can use C<FH> as the filehandle, in C<< close FH >> and C<<
 <FH> >> and so on.  Note that it's a global variable, so this form is
-not recommended when dealing with filehandles other than Perl's built-in ones (e.g. STDOUT and STDIN).
+not recommended when dealing with filehandles other than Perl's built-in ones
+(e.g. STDOUT and STDIN).  In fact, using a bareword for the filehandle is
+I<illegal> when the C<bareword_filehandles> feature has been disabled.  This
```
Comment (Grinnz) [inline pod/perlfunc.pod]: We don't often use the term "illegal" for this; perhaps "an error" or "a fatal error" or "a compilation error".

**Comment by leonerd** [comment]
Code-wise LGTM
A couple of small comments from me and Grinnz on the subject of docs formatting/wording but good other than those.

File: pod/perldelta.pod (hunk 1 above)
Comment (rjbs) [reply pod/perldelta.pod]: Sure.

File: pod/perlfunc.pod (hunk 2 above)
Comment (rjbs) [reply pod/perlfunc.pod]: Will update.

**Comment by rjbs** [comment]
Actually what nobody noticed is that (oops) I documented removing bareword filehandles but didn't do it.  I will do this now.

**Comment by rjbs** [comment]
Updated, and tests now running.

**Review by leonerd — APPROVED** [review]
LGTM

**Pull request merged**